README.md

```diff
@@ -56,99 +58,79 @@ $ pod install
-4. Hit the **Save** button in the top left, to save these updates. 
+1. Sign in to the [Azure portal](https://portal.azure.com) using either a work or school account.
+2. In the left-hand navigation pane, select the **Azure Active Directory** blade, and then select **App registrations**.
+3. Click on the **New registration** button at the top left of the page.
+4. On the app registration page,
+   - Name your app
+   - Under **Supported account types**, select **Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)**
+   - Click **Register** to finish.
+5. After the app is created, you'll land on your app management page. Take note of the **Application (client) ID** as this would be needed for the step 1C below.
+6. Click **Authentication**, and add new Redirect URI with type **Public client (mobile & desktop)**. Enter redirect URI in format: `msauth.<app_bundle_id>://auth`. Replace <app_bundle_id> with the **Bundle Identifier** for your application. 
+7. Hit the **Save** button in the top left, to save these updates.
 
-## 1B: Configure your application
+## 2B: Configure your application
+
+1. Update your application's redirect URI scheme in the  `Info.plist` file by replacing `msauth.com.microsoft.identitysample.MSALiOS` . Redirect URI scheme follows the format `msauth.[app_bundle_id]`. Make sure to substitue [app_bundle_id] with the **Bundle Identifier** for your application. 
 
-1. Add your application's redirect URI scheme to added in the portal to your `info.plist` file. It will be in the format of `msauth.<app.bundle.id>`
 ```xml
-    <key>CFBundleURLTypes</key>
+<key>CFBundleURLTypes</key>
+<array>
+  <dict>
+    <key>CFBundleURLSchemes</key>
     <array>
-        <dict>
-            <key>CFBundleTypeRole</key>
-            <string>Editor</string>
-            <key>CFBundleURLName</key>
-            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
-            <key>CFBundleURLSchemes</key>
-            <array>
-                <string>msauth.<app.bundle.id></string>
-            </array>
-        </dict>
+      <string>msauth.com.microsoft.identitysample.MSALiOS</string>
     </array>
+  </dict>
+</array>
 ```
 
 2. Configure your application defaults
 
-In the `ViewControler.swift` file, update the `kClientID` variable with your client ID.
+In the `ViewControler.swift` file, update the `kClientID` variable with your Application (client) ID.
 
 ```swift
-    // Update the below to your client ID you received in the portal. The below is for running the demo only
+    // Update the below with the client ID you received in the portal. 
     
-    let kClientID = "<your-client-id-here>"
+    let kClientID = "66855f8a-60cd-445e-a9bb-8cd8eadbd3fa"
```

Comment:
Just curious, what is the reason by replacing `<your-client-id-here>` with real client id here? :)

Reply:
This is to make it consistent with the Mac sample and also to make sure that it works out of box. Comment above this line asks to replace if the user registers the app.

Reply:
Should we still mention that "The below is for running the demo only" for clarity?

Reply:
Added.